net/instaweb/rewriter/add_instrumentation.js

```diff
@@ -88,21 +88,30 @@ pagespeed.AddInstrumentation.prototype.sendBeacon = function() {
         timingApi['responseEnd'] - timingApi['responseStart']);
     url += '&dom_c=' + (timingApi['domContentLoadedEventStart'] - navStartTime);
 
-    if (window['performance']['navigation']) {
-      url += '&nt=' + window['performance']['navigation']['type'];
+    if (winPerf['navigation']) {
+      url += '&nt=' + winPerf['navigation']['type'];
     }
     var firstPaintTime = -1;
+    var paintEntries, loadTimes;
     if (timingApi['msFirstPaint']) {
       // IE.
       firstPaintTime = timingApi['msFirstPaint'];
-    } else if (window['chrome'] && window['chrome']['loadTimes']) {
+    } else if (winPerf['timeOrigin'] &&
+        winPerf['getEntriesByType'] &&
+        (paintEntries = winPerf['getEntriesByType']('paint')).length > 0) {
+        // Paint Timing API
+        // Note that getEntriesByType('paint') is sorted chronologically
+        firstPaintTime = Math.round(
```

Comment:
can you move the var declaration for firstPaintTime outside the if-block so it's obvious that scopes to all the branches?